docker-compose.circleci.yml

```diff
@@ -44,6 +44,13 @@ services:
       - MINIO_ACCESS_KEY=minioadmin
       - MINIO_SECRET_KEY=minioadmin
       - MINIO_HOST=minio:9000
+      - PGHOST=db
+      - PGPORT=5432
+      - PGDATABASE=postgres
+      - PGUSER=postgres
+      - PGPASS=mysecretpassword
```

Comment:
Should this password be in a more hidden config for cypress? I don't think we'd leak anything, but want to double check.

Reply:
We have it in [other docker compose files already](https://github.com/CMSgov/easi-app/blob/master/docker-compose.cypress_local.yml#L22), so I'm assuming it's OK to have it here as well.